aspnet/web-api/overview/security/individual-accounts-in-web-api/samples/sample5.cmd

```diff
@@ -7,4 +7,4 @@ X-Requested-With: XMLHttpRequest
 Referer: https://localhost:44305/
 Content-Length: 84
 
-{"Email":"alice@example.com","Password":"Password1!","ConfirmPassword":"Password1!"}
\ No newline at end of file
+{"Email":"alice@example.com","Password":$CREDENTIAL_PLACEHOLDER$,"ConfirmPassword":"$CREDENTIAL_PLACEHOLDER$"}
```

Comment:
```suggestion
{"Email":"alice@example.com",$CREDENTIAL_PLACEHOLDER`$,$CREDENTIAL_PLACEHOLDER2$"}
```

Reply:
Thanks, yes, I couldn't see how to handle this any better with what's there to work with.  I was trying to avoid spiraling into much more major rework of the doc and samples unless we really needed to.